ecma402/stage-0-proposals.md

```diff
@@ -9,11 +9,9 @@ Stage 0 proposals are either
 | -------------------------------------------------------------------- | ---------------- | ------------------ |
 | [Fix 9.2.3 LookupMatcher algorithm][]                                | Rafael Xavier    | Rafael Xavier      |
 | [`Intl.NumberFormat` `round` option][intl.numberformat round option] | Rafael Xavier    | Rafael Xavier      |
-| [`Intl.LocaleMatcher`][intl-localematcher]                           | Long Ho          | Long Ho            |
```

Comment:
ecma402 stage 1 include the proposals 
see https://github.com/tc39/proposals/tree/master/ecma402#stage-1

Reply:
CC @ljharb